tensorflow_quantum/python/optimizers/rotosolve_minimizer.py

```diff
@@ -0,0 +1,273 @@
+# Copyright 2020 The TensorFlow Quantum Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""The rotosolve minimization algorithm"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+import math

```

Comment:
math.pi is used here but np.pi is used in the tests. Could we change this module to also use np.pi just for consistency ?

Reply:
Fixed in e9da877e4b3be1d187164f92a2fcf6aba0def909